editor/components/default-block-appender/style.scss

```diff
@@ -11,6 +11,39 @@ $empty-paragraph-height: $text-editor-font-size * 4;
 			outline: 1px solid $light-gray-500;
 		}
 	}
+
+	.editor-inserter-with-shortcuts {
```

Comment:
Not sure what of it is related to changes here, but default inserter does some unexpected things at smaller viewports:

![image](https://user-images.githubusercontent.com/1779930/37104547-ff556c1a-21fa-11e8-88f4-7e8443416776.png)